### ISSUE TITLE
Slider missing home/end accessibility shortcuts

### ISSUE THREAD

**Issue by dbjorge** [microsoft/fast-dna#1914]
**Describe the bug; what happened?**

[Slider's README Accessibility section](https://github.com/microsoft/fast-dna/tree/master/packages/fast-components-react-base/src/slider#accessibility) documents:

> Slider implements the recommended keyboard navigation scheme described [here](http://w3c.github.io/aria-practices/examples/slider/multithumb-slider.html).

However, Slider is missing implementations for 2 of the recommended keyboard shortcuts: "home" or "end" should ideally set the focused slider to its minimum/maximum value.

**What are the steps to reproduce the issue?**

1. Go to https://explore.fastdna.net/components/slider/examples/
2. Tab to any of the sliders
3. Observe that while left/right/pgdn/pgup have the expected behavior, home/end don't have any special behavior

**What behavior did you expect?**

home/end should minimize/maximize the slider value, per [WAI-ARIA Authoring Practices 1.1 §3.19](http://w3c.github.io/aria-practices/#keyboard-interaction-17) ([example](http://w3c.github.io/aria-practices/examples/slider/multithumb-slider.html))

**Comment by chrisdholt** [comment]
Thanks @dbjorge - looks like this was missed. We’ll get it updated.

**Comment by chrisdholt** [comment]
@scomea do you want to knock this one out? I think the links we want are:
- http://w3c.github.io/aria-practices/#slider
- http://w3c.github.io/aria-practices/#slidertwothumb
